### PR TITLE
bump quarkus, kogito and camel

### PR DESCRIPTION
**Pull request by ricardozanini**
@spolti, @kmacedovarela  do you want to take a look?

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -41,7 +39,7 @@ build-s2i-weather-image:
 .PHONY: build-s2i-rain-image
 build-s2i-rain-image:
 	@echo .......... Building S2I Rain Forecast Process Image ...............
-	s2i build rain-forecast-process/ quay.io/kiegroup/kogito-quarkus-ubi8-s2i:0.4.0 quay.io/${QUAY_NAMESPACE}/rain-forecast-process:${VERSION}  -e NATIVE=true --runtime-image quay.io/kiegroup/kogito-quarkus-ubi8:0.4.0
+	s2i build rain-forecast-process/ quay.io/kiegroup/kogito-quarkus-ubi8-s2i:0.5.0 quay.io/${QUAY_NAMESPACE}/rain-forecast-process:${VERSION}  -e NATIVE=true --runtime-image quay.io/kiegroup/kogito-quarkus-ubi8:0.5.0
```
Comment (spolti) [inline Makefile]: use 0.5.1 images?

File: openshift/rain-forecast/operator/rain-forecast-kogitoapp.yaml [hunk 2]
```diff
@@ -8,9 +8,9 @@ spec:
       uri: https://github.com/ricardozanini/tenkichannel 
       contextDir: rain-forecast-process
     imageRuntime:
-      imageStreamTag: 0.4.0
+      imageStreamTag: 0.5.0
```
Comment (spolti) [inline openshift/rain-forecast/operator/rain-forecast-kogitoapp.yaml]: 0.5.1?

File: rain-forecast-process/Dockerfile.kogito [hunk 3]
```diff
@@ -1,5 +1,6 @@
-FROM quay.io/kiegroup/kogito-quarkus-ubi8:0.4.0
+FROM quay.io/kiegroup/kogito-quarkus-ubi8:0.5.0
```
Comment (spolti) [inline rain-forecast-process/Dockerfile.kogito]: 0.5.1?

File: weather-api-gateway/pom.xml [hunk 4]
```diff
@@ -14,8 +14,8 @@
         <maven.compiler.target>1.8</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <!-- update this version when Camel update theirs -->
-        <version.io.quarkus>0.22.0</version.io.quarkus>
-        <version.org.apache.camel.quarkus>0.2.0</version.org.apache.camel.quarkus>
+        <version.io.quarkus>1.0.0.CR1</version.io.quarkus>
```
Comment (spolti) [inline weather-api-gateway/pom.xml:17]: version 1.0.0.Final is released already.

File: weather-api-gateway/src/main/java/org/tenkichannel/weather/api/gateway/resources/WeatherResource.java [hunk 5]
```diff
@@ -1,19 +1,14 @@
...
-import org.apache.camel.quarkus.core.runtime.CamelRuntime;
-import org.tenkichannel.weather.api.gateway.routes.WeatherGatewayRoute;
+import org.apache.camel.CamelContext;
 import org.tenkichannel.weather.api.gateway.domain.Forecast;
 import org.tenkichannel.weather.api.gateway.domain.Location;
 import org.tenkichannel.weather.api.gateway.domain.Weather;
+import org.tenkichannel.weather.api.gateway.routes.WeatherGatewayRoute;
+
+import javax.inject.Inject;
+import javax.ws.rs.*;
```
Comment (spolti) [inline weather-api-gateway/src/main/java/org/tenkichannel/weather/api/gateway/resources/WeatherResource.java]: IMHO use * on imports is not a good practice.

**Review by spolti — CHANGES_REQUESTED** [review]
small tweaks, looks like the camel guys have improved the camel config :)

File: weather-api-gateway/pom.xml (hunk 4 above)
Comment (ricardozanini) [reply weather-api-gateway/pom.xml:17]: But Camel hasn’t updated their side yet :(

File: weather-api-gateway/src/main/java/org/tenkichannel/weather/api/gateway/resources/WeatherResource.java (hunk 5 above)
Comment (ricardozanini) [reply weather-api-gateway/src/main/java/org/tenkichannel/weather/api/gateway/resources/WeatherResource.java]: Dammit IntelliJ. Do you know how do I fix that?

Comment (ricardozanini) [reply weather-api-gateway/src/main/java/org/tenkichannel/weather/api/gateway/resources/WeatherResource.java]: https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html

**Review by spolti — APPROVED** [review]

**Pull request merged**